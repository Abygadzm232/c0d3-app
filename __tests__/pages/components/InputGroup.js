import React from 'react'
import renderer from 'react-test-renderer'
import InputGroup from '../../../components/InputGroup'

describe('Snapshot for InputGroup', () => {
  test('Small InputGroup', () => {
    const component = renderer.create(<InputGroup size="small" />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

  test('Default InputGroup', () => {
    const component = renderer.create(<InputGroup />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

  test('Large InputGroup', () => {
    const component = renderer.create(<InputGroup size="large" />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

  test('Large InputGroup', () => {
    const component = renderer.create(<InputGroup isDisabled />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
