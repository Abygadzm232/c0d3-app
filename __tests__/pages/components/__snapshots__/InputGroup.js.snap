// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot for InputGroup Default InputGroup 1`] = `
<div
  className="input-group  col-6"
>
  <input
    aria-describedby="basic-addon2"
    aria-label="email"
    className="form-control"
    disabled={false}
    placeholder=""
    type="text"
  />
  <div
    className="input-group-append"
  >
    <button
      className="btn btn-primary"
      disabled={false}
      id="button-addon2"
      type="button"
    />
  </div>
</div>
`;

exports[`Snapshot for InputGroup Large InputGroup 1`] = `
<div
  className="input-group input-group-lg col-6"
>
  <input
    aria-describedby="basic-addon2"
    aria-label="email"
    className="form-control"
    disabled={false}
    placeholder=""
    type="text"
  />
  <div
    className="input-group-append"
  >
    <button
      className="btn btn-primary"
      disabled={false}
      id="button-addon2"
      type="button"
    />
  </div>
</div>
`;

exports[`Snapshot for InputGroup Large InputGroup 2`] = `
<div
  className="input-group  col-6"
>
  <input
    aria-describedby="basic-addon2"
    aria-label="email"
    className="form-control"
    disabled={true}
    placeholder=""
    type="text"
  />
  <div
    className="input-group-append"
  >
    <button
      className="btn btn-primary"
      disabled={true}
      id="button-addon2"
      type="button"
    />
  </div>
</div>
`;

exports[`Snapshot for InputGroup Small InputGroup 1`] = `
<div
  className="input-group input-group-sm col-6"
>
  <input
    aria-describedby="basic-addon2"
    aria-label="email"
    className="form-control"
    disabled={false}
    placeholder=""
    type="text"
  />
  <div
    className="input-group-append"
  >
    <button
      className="btn btn-primary"
      disabled={false}
      id="button-addon2"
      type="button"
    />
  </div>
</div>
`;
